packages/py3dns/PKGBUILD

```diff
@@ -0,0 +1,28 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=python-dns
+_pkgname=py3dns
+pkgver=3.2.1
+pkgrel=1
+pkgdesc='A module for looking up DNS entries in Python applications.'
+arch=('any')
+url='https://pypi.org/project/py3dns/'
+license=('Python')
```

Comment:
Python License (CNRI Python License) is not in `/usr/share/licenses/common/`

please follow https://wiki.archlinux.org/title/PKGBUILD#license

Reply:
thanks, totally forgot it :))